module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,57 @@
+#include "compression.hpp"
+
+#include <iostream>
+
+void printMap(const std::array<std::array<uint8_t, width>, height>& bitmap) {
+
+    for (const auto& arr : bitmap) {
+        for (const auto& el : arr) {
+            if (el <= ' ')
+                std::cout << " ";
+            else
+                std::cout << el;
+        }
+        std::cout << "\n";
+    }
+}
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(const std::array<std::array<uint8_t, width>, height>& grayScale) {
+    std::vector<std::pair<uint8_t, uint8_t>> compressed;
+    compressed.reserve(width);
+    int count = 1, tmp;
```

Comment:
one declaration/ definition per line. Please initialize tmp value.

Reply:
If we are sure that before we use it we will initialize it anyway, we must do it in advance?